classes/fields/currency.php

```diff
@@ -312,7 +312,7 @@ public function display( $value = null, $name = null, $options = null, $pod = nu
 		// Multiple sign currencies: 100 Fr, Kr 100
 		$currency_gap = '';
 
-		if ( strlen( $currency_sign ) > 1 && 0 !== strpos( $currency_sign, '&' ) ) {
+		if ( strlen( $currency_sign ) > 1 && false === strpos( $currency_sign, '&' ) ) {
```

Comment:
I think we're safe, if someone has issues we can always approach that separately when the time comes.